calico-cloud_versioned_docs/version-3.16/image-assurance/understanding-scan-results.mdx

```diff
@@ -2,111 +2,97 @@
-You can change the CVSS score threshold between a Pass and a Warn, and between a Warn and a Fail. Currently, only [CVSS v3 scores](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator) are displayed. CVEs that do not have a CVSS v3 score (too old to have one, or are too new to be assigned one), display a blank score in the UI, and **N/A** in the CLI.
 
-| CVSS v3 scores                              | Image Assurance mapping | Default settings     |
-| ------------------------------------------- | ----------------------- | -------------------- |
-| 0.0 – (None)<br />0.1 – 3.9 (Low)           | Pass = 3.9              | Low                  |
-| 4.0 – 6.9 (Medium)                          | Warn = 7                | Medium severity – 7  |
-| 7.0 – 8.9 (High)<br />9.0 – 10.0 (Critical) | Fail = 8.0              | Critical or high – 8 |
+**Changing the CVSS thresholds**
 
-**Changing the default CVSS threshold values**
+On the right panel, click the **Settings** icon. When changing the CVSS threshold values, note the following:
+- Changes to values take effect immediately, and alter the scan results for images already in the system
+- If you are using admission controller policies, changing a value may allow pods in your Kubernetes cluster that were previously blocked, to now be deployed or vice versa.
 
-The default threshold values (shown below) will work for the majority of {{prodname}} deployments. However, you may need to change the defaults because of security requirements.
+**Why do I have images with `unknown` results**?
 
-![scan-settings](/img/calico-cloud/scan-settings.png)
+If scan results contain `unknowns` it can mean:
+- The Image Assurance scanner is not enabled and running
+- The scanner has not yet reported results yet
 
-To change the CVSS threshold values, note the following:
+**How long does it take for scan results?**
 
-- Changes to threshold values take effect immediately and alter the scan results for images already in the system
-- If you are using admission controller policies, changing a value may allow pods in your Kubernetes cluster that were previously being blocked, to now be deployed or vice versa.
+Anywhere from 2-10 minutes. The variability occurs because the scanner watches, reports, and then sleeps to balance efficiency and resources.
 
-**Enable Runtime View**
+**What is Download Scanner?**
 
-You can configure your scan settings to directly monitor pods that are running in your Kubernetes clusters.
-When enabled, Runtime View checks image information from the active container images in all of your connected clusters.
-These images are correlated with the images you have recorded in the Image Assurance system.
+This link goes to steps to integrate the Image Assurance scanner as part of your build pipeline. If you want to scan images only in your Kubernetes cluster, see [Enable automatic scanning inside a cluster](./scanners/cluster-scanner).
```

Comment:
I'm finding a bit hard to follow which "scanner" we're talking about throughout this page, since it's switching quite a bit from section to section about which scanner it's talking about. 

Here it is talking about the CLI scanner. But earlier on in this page it was referring to the cluster scanner as the "scanner".